rllib/agents/trainer_config.py

```diff
@@ -902,15 +902,23 @@ def offline_data(
                 when the offline file has been generated by another RLlib algorithm
                 (e.g. PPO or SAC), while "normalize_actions" was set to True.
             input_evaluation: DEPRECATED: Use `off_policy_estimation_methods` instead!
-            off_policy_estimation_methods: Specify how to evaluate the current policy.
+            off_policy_estimation_methods: Specify how to evaluate the current policy,
+                along with any optional config parameters.
                 This only has an effect when reading offline experiences
                 ("input" is not "sampler").
-                Available options:
-                - "simulation": Run the environment in the background, but use
+                Available keys:
+                - {"simulation": None}: Run the environment in the background, but use
                 this data for evaluation only and not for learning.
-                - Any subclass of OffPolicyEstimator, e.g.
-                ray.rllib.offline.estimators.is::ImportanceSampling or your own custom
-                subclass.
+                - {ope_method_name: {"type": ope_type, ...}} where `ope_type` can be:
```

Comment:
This is great! Can you add that `ope_method_name` may be any user-defined string?